src/views/RoomSelector.vue

```diff
@@ -66,26 +74,13 @@ export default {
 			rooms: [],
 			selectedRoom: null,
 			loading: true,
-			// TODO: should be included once this is properly available
-			types: {
-				ROOM_TYPE_ONE_TO_ONE: 1,
-				ROOM_TYPE_GROUP: 2,
-				ROOM_TYPE_PUBLIC: 3,
-				ROOM_TYPE_CHANGELOG: 4,
-			},
 		}
 	},
 	computed: {
-		currentRoom() {
-			if (OCA.SpreedMe && OCA.SpreedMe.app.activeRoom) {
-				return OCA.SpreedMe.app.activeRoom.get('token')
-			}
-			return null
-		},
 		availableRooms() {
 			return this.rooms.filter((room) => {
-				return room.token !== this.currentRoom
-					&& room.type !== this.types.ROOM_TYPE_CHANGELOG
+				return room.readOnly !== CONVERSATION.STATE.READ_ONLY
```

Comment:
I assume this change is due to the reuse of the collaboration API, but this would allow to link a talk room to itself, where i'm not sure what happens when you try to do that in the backend.

Reply:
we don't use the RoomSelector within Talk, so not sure where else it's being used ?

but as you say it might be safer to also allow read-only rooms still

Reply:
adjusted to also allow listing read-only rooms